### PR TITLE
Update ProfileTOS to fix grammar

### PR DESCRIPTION
**Pull request by wrac4242**

### CODE REVIEW

File: ProfileTOS.md [hunk 1]
```diff
@@ -26,25 +26,30 @@ Because we can and want you to know what we save and do with your information.
...
 The above information is publicly visible by visiting our [stats page](https://stats.founntain.dev) or by viewing profiles inside the client.
 
-Thats all the data we store of you in our *database*, when you delete your profile the above object of you is gone and erased.  
-Your profile picture will also be deleted when you delete your profile, obviously.
+That's all the data we store of you in our *database*, when you delete your profile the above object of you is gone and erased.  
+Your profile picture will also be deleted when you delete your profile.
 
 ### ☝️ Some rules
-Don't post anything NSFW, hate-/harmful in your description or somewhere else in the client. This also includes your profile picture. *(sorry about that, but no anime tiddies as profile picture)*  
+Don't post anything NSFW, hate-/harmful in your description or somewhere else in the client. This also includes your profile picture. *(sorry about that, but no anime titties as profile picture)*
```
Comment (Founntain) [inline ProfileTOS.md:42]: Someone doesn't like Tiddies :(

File: ProfileTOS.md [hunk 2]
```diff
@@ -26,25 +26,30 @@ Because we can and want you to know what we save and do with your information.
...
 
-If you create a profile and don't reach level 2 in 24 hours **we have the right to delete your profile**. Please only create on if you intend to use the player and its user feature aswell and as intended.  
-You don't need a profile if you don't use the user feature to collect XP and statistics! If you profile was deleted, you will still be able to create a new profile if you think you use it as intended.
+If you create a profile and don't reach level 2 in 24 hours **we have the right to delete your profile**. Please only create an account if you intend to use the player and its user feature as intended.  
+You don't need a profile if you do not want to use the user features, such as collecting XP and statistics! If your profile was deleted, you will still be able to create a new profile if you believe you will use it as intended.
 
-⚠️ If for whatever reason you do so, we have the right to clear your description, remove your profile picture / banner or send your profile to the *sun (deleting it)*.
+## 📫 Contact us
+If you wish to contact us for any reason please contact us using the links below
+- ✉️ 7@founntain.dev
```
Comment (Founntain) [inline ProfileTOS.md:49]: Maybe add a mailto: link ?

File: ProfileTOS.md [hunk 3]
```diff
@@ -26,25 +26,30 @@ Because we can and want you to know what we save and do with your information.
...
 
-If you create a profile and don't reach level 2 in 24 hours **we have the right to delete your profile**. Please only create on if you intend to use the player and its user feature aswell and as intended.  
-You don't need a profile if you don't use the user feature to collect XP and statistics! If you profile was deleted, you will still be able to create a new profile if you think you use it as intended.
+If you create a profile and don't reach level 2 in 24 hours **we have the right to delete your profile**. Please only create an account if you intend to use the player and its user feature as intended.  
+You don't need a profile if you do not want to use the user features, such as collecting XP and statistics! If your profile was deleted, you will still be able to create a new profile if you believe you will use it as intended.
 
-⚠️ If for whatever reason you do so, we have the right to clear your description, remove your profile picture / banner or send your profile to the *sun (deleting it)*.
+## 📫 Contact us
```
Comment (Founntain) [inline ProfileTOS.md:47]: Maybe swap the contact use with the warning from beloe. so warning first anf then contact

**Review by Founntain — COMMENTED** [review]
Want to hear your thoughs about my comments.

File: ProfileTOS.md (hunk 2 above)
Comment (wrac4242) [reply ProfileTOS.md:49]: This was copied and pasted off the readme page, will update that too if you want

Comment (Founntain) [reply ProfileTOS.md:49]: Yep would be nice

File: ProfileTOS.md (hunk 3 above)
Comment (wrac4242) [reply ProfileTOS.md:51]: done

File: ProfileTOS.md (hunk 2 above)
Comment (wrac4242) [reply ProfileTOS.md]: done

File: ProfileTOS.md (hunk 1 above)
Comment (wrac4242) [reply ProfileTOS.md]: changed back

**Review by Founntain — APPROVED** [review]

**Pull request merged**